lsp/nls/src/field_walker.rs

```diff
@@ -200,6 +227,28 @@ impl<'a> FieldResolver<'a> {
         fields
     }
 
+    pub fn get_cousin_defs(&self, def: &DefWithPath) -> Vec<(LocIdent, Field)> {
+        let mut ret = Vec::new();
+        if let Some(parent) = &def.parent_record {
+            if let Some(mut ancestors) = self.server.analysis.get_parent_chain(parent) {
+                while let Some(ancestor) = ancestors.next_merge() {
+                    // We're traversing up the tree starting at the parent, so this is the
+                    // path to the parent (not the original def).
+                    if let Some(parent_path) = ancestors.path() {
+                        let uncles = self.resolve_term_path(&ancestor, parent_path.iter().copied());
```

Comment:
So the uncles are the other leaf of the merge expression enclosing the current term? I think those method could benefit from being described at a high level in the documentation (even more if they're public API, despite "public API" being a bit less meaningful for NLS)